docs/book/component-guide/model-deployers/mlflow.md

```diff
@@ -7,7 +7,7 @@ description: Deploying your models locally with MLflow.
 The MLflow Model Deployer is one of the available flavors of the [Model Deployer](./) stack component. Provided with the MLflow integration it can be used to deploy and manage [MLflow models](https://www.mlflow.org/docs/latest/python_api/mlflow.deployments.html) on a local running MLflow server.
 
 {% hint style="warning" %}
-The MLflow Model Deployer is not yet available for use in production. This is a work in progress and will be available soon. At the moment it is only available for use in a local development environment.
+The MLflow Model Deployer is only meant for deploying models in development settings, for production model deployment we recommend tools like [BentoML](./bentoml.md) or [Seldon](./seldon.md).
```

Comment:
```suggestion
The MLflow Model Deployer is not meant for deploying models in development settings, for production model deployment we recommend tools like [BentoML](./bentoml.md) or [Seldon](./seldon.md).
```

Reply:
no, it is ONLY meant ... DEVELOPMENT ..., for PRODUCTION ... we recommend ...